tests/test.ts

```diff
@@ -18,6 +18,7 @@ describe('This is to create a club on firebase', () => {
       name: 'ACM',
       description: 'Computer science club at UTD',
       contacts: { email: 'deadmail@deadmail.com' },
+      id: 'test',
```

Comment:
We create a separate ID field so that the object itself contains a reference to its own ID since firestore defines the ID differently than normal databases